### ISSUE TITLE
Calling (figwheel.main/reset) on a merged build fails

### ISSUE THREAD

**Issue by lyderichti59** [bhauman/figwheel-main#268]
Hi, 
I think I may have spotted an issue when resetting a composite build.

Let `dev1.cljs.edn` and `dev2.cljs.edn` be two build files and let say I want to merge their options for my Figwheel REPL session.
I can start the REPL this way :
`clojure -m figwheel.main --build dev1:dev2 --repl`
As mentioned in the doc, multiple build names are separated by a ':'.

A REPL session is launched with the correct merged options. When I execute `(figwheel.main/status)`, I can see that the build id available and auto-built is "dev1dev2".

Let say I modify one or both the "dev{1,2}.cljs.edn" files and I want to reset my config.
I call directly from the REPL `(figwheel.main/reset)` but this happens :
```
Cleaning build id: dev1dev2
Stopped building id: dev1dev2
Reloading config for id: dev1dev2
Compile Exception: Build dev1dev2.cljs.edn does not exist
```

Is there a proper way to reset a composite build ?

Hint : I just thought it might be possible to keep the "original" composite build id (with the ':' separator) rather than generating an unparsable one. If it is possible, then everywhere a build id list is provided, if one of the id has a ":" sign, it must be considered as a composite build and use appropriate actions.

What's your opinion on this ?

Thanks for the work you do !

**Comment by bhauman** [comment]
Yes reset doesn't support merged builds right now.  It makes sense that it should but it's kinda low priority.

If I was going to do it I propagate meta information about the where the build came from so that it can rebuilt it properly.
